.github/workflows/build_test.sh

```diff
@@ -84,6 +84,14 @@ if [[ "$COMPILER" == clang ]]; then
     CXX="clang++-$COMPILER_VERSION"
     AR="llvm-ar-$COMPILER_VERSION"
 
+    if systemd-analyze compare-versions "$COMPILER_VERSION" ge 17; then
+        CFLAGS="-fno-sanitize=function"
+        CXXFLAGS="-fno-sanitize=function"
```

Comment:
I don't think systemd is built with UBSan in this workflow so I'm not sure why this is needed. If it's necessary I think it would be better to pass `c_args` and so on so that it was consistent.

Reply:
If I remember correctly it was building the fuzz tests that broke without this.

Reply:
It should be fixed in `meson.build` responsible for building those fuzz targets then.

Reply:
Either way that action switched to clang-18 in 4651e1428dae935bb32dec082db5155f2e8b1c98 and systemd didn't fail to compile so I have to admit I'm not sure why it would fail here.

Reply:
I was curious so I reverted the CFLAGS part in my fork and pushed it to see what can possibly fail: https://github.com/evverx/systemd/actions/runs/9435948677/job/25990022282?pr=114.